test/integration/test_ibm_job.py

```diff
@@ -139,6 +139,50 @@ def test_retrieve_jobs(self):
         for job in job_list:
             self.assertTrue(isinstance(job.job_id(), str))
 
+    def test_retrieve_jobs_with_status(self):
+        """Test retreiving jobs with status filter."""
+        statuses = [["DONE"], JobStatus.DONE, [JobStatus.DONE]]
+        job_list = self.provider.backend.jobs(
+            backend_name=self.sim_backend.name, limit=5, status="DONE"
+        )
+        for status in statuses:
+            new_list = self.provider.backend.jobs(
+                backend_name=self.sim_backend.name, limit=5, status=status
+            )
+            self.assertEqual(
+                [job.job_id() for job in job_list], [job.job_id() for job in new_list]
+            )
+
+        completed_job_list = self.provider.backend.jobs(
+            backend_name=self.sim_backend.name, limit=5, status="completed"
+        )
+        for job in completed_job_list:
+            self.assertTrue(
+                job.status() in [JobStatus.DONE, JobStatus.CANCELLED, JobStatus.ERROR]
+            )
```

Comment:
Can we put each `jobs()` call into its own test case? So they get run even if a previous call fails.